skiko/src/jvmMain/cpp/common/interop.cc

```diff
@@ -1202,3 +1200,36 @@ namespace skija {
         }
     }
 }
+
+namespace kotlin {
+    namespace jvm {
+        namespace functions {
+            namespace Function0 {
+                jclass cls;
+                jmethodID invoke;
+
+                void onLoad(JNIEnv* env) {
+                    jclass local = env->FindClass("kotlin/jvm/functions/Function0");
+                    cls   = static_cast<jclass>(env->NewGlobalRef(local));
```

Comment:
Extra space.

Reply:
Let's perform style fixed in a separate PR, there at least 3 places with same issue